code/mobile/mobile-common/PVRRenderer.cpp

```diff
@@ -34,9 +34,19 @@ namespace {
 
     Matrix4f gvrToEigenMat(Mat4f gvrMat) {
         Matrix4f eMat;
-        for (int i = 0; i < 4; ++i)
-            for (int j = 0; j < 4; ++j)
-                eMat(i, j) = gvrMat.m[i][j];
+        try {
+            for (int i = 0; i < 4; ++i) {
+                for (int j = 0; j < 4; ++j) {
+                    eMat(i, j) = gvrMat.m[i][j];
+                }
+            }
+        }
+        catch(exception e) {
+            PVR_DB("PVRRenderer_gvrToEigenMat:: Caught Exception: " + string(e.what()));
+        }
+        catch(...) {
```

Comment:
Still many Generic Catchs are implemented, and many exception messages are PVR_DB instead of PVR_DB_I